### DISCUSSION TITLE
How do I get the project generated api TOC to display on the page created by index.md?

### DISCUSSION THREAD

**Discussion by woutware** [dotnet/docfx#10142] [Q&A]
I have a Visual Studio solution that generates html with docfx. I added an index.md home page, which works fine, but I want the generated api TOC with all the namespaces of the VS solution to show on the left, like it does on all the generated pages. So I don't want to create my own TOC to associate with my index.md page, I just want the generated api TOC to be displayed.

**Comment by icnocop** [comment]
I was able to "resolve" this by redirecting `index.html` to a different page which already has the TOC.

index.md:
```
---
redirect_url: "articles/Introduction.html"
---
```

In this way, the table of contents (TOC) is displayed on the left side, as they'll normally see it on other pages, and it can also be filtered.

I'm using the `modern` template.

Hope that helps, even though it's not exactly what you wanted. 😊
